src/config.tsx

```diff
@@ -0,0 +1,27 @@
+type EnvConfig = {
+  readonly sentry: {
+    dsn: string;
+  };
+};
+
+type Config = {
+  readonly development: EnvConfig;
+  readonly production: EnvConfig;
+};
+
+const config: Config = {
+  development: {
+    sentry: {
+      dsn: ""
+    }
+  },
+  production: {
+    sentry: {
+      dsn:
+        "https://a4537e18a1014f908068985f7dc12609@o391523.ingest.sentry.io/5240240"
+    }
+  }
+};
```

Comment:
This should be enough to type the config:
```ts
const config = {
  development: { ... },
  production: { ... }
} as const
```

Reply:
I wanted to make sure all the configs for each environment had the same attributes, this wouldn't check for that right?

Reply:
Nope, that wouldn't check that, then leave it as it is.